packages/sw-cli/src/lib/templates/sw.js.tmpl

```diff
@@ -0,0 +1,6 @@
+importScripts(
+  "<%= manifestPath %>",
```

Comment:
Changes to files listed in `importScripts` won't trigger service worker updates yet. (The [tracking bug](https://bugs.chromium.org/p/chromium/issues/detail?id=648295) says that it's targeted for Chrome 57, but I think the branch for that happened already, so I assume Chrome 58 is more likely.)

Therefore, we need to inline the manifest file if we want changes to the manifest to trigger the `install` events.